lib/font_awesome_flutter.dart

```diff
@@ -2,6 +2,7 @@ library font_awesome_flutter;
 
 import 'package:flutter/widgets.dart';
 import 'package:font_awesome_flutter/icon_data.dart';
+export 'fa_icon.dart';
```

Comment:
This line needs to be added to the [generator](https://github.com/fluttercommunity/font_awesome_flutter/blob/c2044e2984ba52c19356049714e89dc1228bc687/tool/generate_font.dart#L54) as well